jdaviz/configs/cubeviz/plugins/tests/test_regions.py

```diff
@@ -69,8 +69,9 @@ def test_spatial_spectral_mix(self):
         # https://github.com/spacetelescope/jdaviz/issues/1584
         with pytest.warns(UserWarning, match='Applying the value from the redshift slider'):
             spectral_subsets = self.cubeviz.specviz.get_spectra()
-        assert list(spectral_subsets.keys()) == ['has_microns[FLUX]',
-                                                 'has_microns[FLUX] (Subset 1)',
-                                                 'has_microns[FLUX] (Subset 2)'], spectral_subsets  # noqa
+        assert list(spectral_subsets.keys()) == ['Spectrum (sum)',
+                                                 'Spectrum (Subset 1, sum)',
+                                                 'Spectrum (sum) (Subset 2)',
+                                                 'Spectrum (Subset 1, sum) (Subset 2)']

```

Comment:
The auto-spectral extraction is creating the entries for `Spectrum (sum)` (on load, for the entire cube) and `Spectrum (<spatial_subset>, sum)` for any created spatial subset using the default labeling logic in spectral extraction (this logic can be up for debate).  `get_spectra()` is then appending the spectral subsets on the end to apply the mask - this behavior is unchanged in this PR.

Reply:
Maybe either the user doc can explain or the label can be less confusing. A new user won't know the difference between "Something (sum) (Subset N)" and "Something( Subset N, sum)" and so on.

Reply:
I think in a real-case, the context of what subsets are what definitely helps.  But I'm open to proposals for changing the default labeling pattern.  We even could change the labeling within `get_spectra` since this is in a major release.